### DISCUSSION TITLE
unifi discovery

### DISCUSSION THREAD

**Discussion by mth3r** [netalertx/NetAlertX#823] [Q&A]
Since unifi went to two factor login, is there a way to use the unifi discovery plug in?  I have a cloud key and a USG.  if I give it the IP of the USG and the unsername and password, it keeps giving me an error.  Any ideas?

**Comment by jokob-sk** [comment]
This should still work, 2FA is not required for API access https://github.com/jokob-sk/NetAlertX/issues/754
